step-templates/deploy-child-project.json

```diff
@@ -1,262 +1,273 @@
ntCanProceed -releaseToDeploy $releaseToDeploy -specificMachineDeployment $specificMachineDeployment -environmentName $environmentName -childDeploymentSpecificMachines $childDeploymentSpecificMachines -project $project -ignoreSpecificMachineMismatch $ignoreSpecificMachineMismatch -deploymentMachines $deploymentMachines -releaseHasAlreadyBeenDeployed $releaseHasAlreadyBeenDeployed -isPromotionMode $isPromotionMode\n\nif ($saveReleaseNotesAsArtifact -eq $true) {\n    $releaseNotes | Out-File 'ReleaseNotes.txt'\n    $currentDate = Get-Date\n    $currentDateFormatted = $currentDate.ToString('yyyy_MM_dd_HH_mm')\n    $artifactName = \"$($project.Name) $($releaseToDeploy.Version) $($sourceDestinationEnvironmentInfo.TargetEnvironment.Name).ReleaseNotes_$($currentDateFormatted).txt\"\n    Write-OctopusInformation \"Creating the artifact $artifactName\"\n    \n    New-OctopusArtifact -Path 'ReleaseNotes.txt' -Name $artifactName\n}\n\nInvoke-RefreshVariableSnapshot -refreshVariableSnapShot $refreshVariableSnapShot -whatIf $whatIf -releaseToDeploy $releaseToDeploy -defaultUrl $defaultUrl -spaceId $spaceId -octopusApiKey $octopusApiKey\n\nif ($whatif -eq $true) {\n    Write-OctopusSuccess 'Exiting because What If set to true.'\n    Write-OctopusInformation \"Setting the output variable ChildReleaseToDeploy to $true\"\n    Set-OctopusVariable -Name 'ChildReleaseToDeploy' -Value $true\n    Exit 0\n}\n\n$approvalTenant = Get-OctopusApprovalTenant -tenantToDeploy $tenantToDeploy -approvalTenantName $approvalTenantName -spaceId $spaceId -defaultUrl $defaultUrl -octopusApiKey $octopusApiKey\n$approvalDeploymentTaskId = Get-ApprovalDeploymentTaskId -autoapproveChildManualInterventions $autoapproveChildManualInterventions -parentDeploymentTaskId $parentDeploymentTaskId -parentReleaseId $parentReleaseId -parentEnvironmentName $parentEnvironmentName -approvalEnvironmentName $approvalEnvironmentName -defaultUrl $defaultUrl -spaceId $spaceId -octopusApiKey $octopusApiKey -parentChannelId $parentChannelId -parentEnvironmentId $parentEnvironmentId -approvalTenant $approvalTenant -parentProject $parentProject\n$parentDeploymentApprovers = Get-ParentDeploymentApprovers -parentDeploymentTaskId $approvalDeploymentTaskId -spaceId $spaceId -defaultUrl $defaultUrl -octopusApiKey $octopusApiKey\n\nCreate-NewOctopusDeployment -releaseToDeploy $releaseToDeploy -targetEnvironment $targetEnvironment -createdDeployment $createdDeployment -project $project -waitForFinish $waitForFinish -enableEnhancedLogging $enableEnhancedLogging -deploymentCancelInSeconds $deploymentCancelInSeconds -defaultUrl $defaultUrl -octopusApiKey $octopusApiKey -spaceId $spaceId -parentDeploymentApprovers $parentDeploymentApprovers -parentProjectName $parentProjectName -parentReleaseNumber $parentReleaseNumber -parentEnvironmentName $approvalEnvironmentName -parentDeploymentTaskId $approvalDeploymentTaskId -autoapproveChildManualInterventions $autoapproveChildManualInterventions -approvalTenant $approvalTenant"
```

Comment:
It looks like this script was completly reformatted with changes like:

* double quotes to single quotes
* curly brackets placed on new lines

The scope of these changes makes it hard to find the logic changes in the script.

Reply:
This is resolved in the latest commit - it was caused by VS Code auto-formatting.